aws/request/offset_reader_test.go

```diff
@@ -32,27 +32,27 @@ func TestOffsetReaderRead(t *testing.T) {
 
 func TestOffsetReaderSeek(t *testing.T) {
 	buf := []byte("testData")
-	reader := newOffsetReader(bytes.NewReader(buf), 0)
+	reader, err := newOffsetReader(bytes.NewReader(buf), 0)
```

Comment:
Should we check for err != nil after func newOffsetReader()  is called?

Reply:
Thanks, missed adding this will, update.